README.md

```diff
@@ -1,33 +1,43 @@
 # Gitness
 Your lightweight, super fast code hosting and continuous integration service.
```

Comment:
Gitness, our new open source development platform packed with the power of code hosting and automated DevOps pipelines